ProteoformSuiteInternal/ResultsSummaryGenerator.cs

```diff
@@ -373,23 +425,115 @@ public static DataTable results_dataframe(TusherAnalysis analysis)
-                    e.modified_mass - e.linked_proteoform_references.Last().modified_mass,
+                    String.Join(", ", (e.linked_proteoform_references.First() as TheoreticalProteoform).ExpandedProteinList.SelectMany(p => p.AccessionList.Select(a => a.Split('_')[0])).Distinct()),
+                    e.ptm_set.ptm_combination.Count == 0 ?
+                        "Unmodified" :
+                        String.Join("; ", e.ptm_set.ptm_combination.Select(ptm => Sweet.lollipop.theoretical_database.unlocalized_lookup.TryGetValue(ptm.modification, out UnlocalizedModification x) ? x.id : ptm.modification.id).OrderBy(m => m)),
+                    e.begin + " to " + e.end,
+                    e.mass_error,
+                    e.modified_mass,
                     e.agg_rt,
                     e.agg_intensity,
-                    get_tusher_values(e.quant, analysis).numeratorIntensitySum,
-                    get_tusher_values(e.quant, analysis).denominatorIntensitySum,
+                    e.topdown_id,
+                    e.topdown_id? (e as TopDownProteoform).correct_id.ToString() : "N/A",
+                    e.ambiguous,
+                    e.adduct,
+                    (e.linked_proteoform_references.First() as TheoreticalProteoform).contaminant,
+                    e.family != null ? e.family.family_id.ToString() : "",
+                    Sweet.lollipop.significance_by_log2FC ? e.quant.Log2FoldChangeValues.numeratorIntensitySum : get_tusher_values(e.quant, analysis).numeratorIntensitySum,
+                    Sweet.lollipop.significance_by_log2FC ? e.quant.Log2FoldChangeValues.denominatorIntensitySum : get_tusher_values(e.quant, analysis).denominatorIntensitySum,
                     Sweet.lollipop.significance_by_log2FC ? e.quant.Log2FoldChangeValues.significant : get_tusher_values(e.quant, analysis).significant
                 );
             }
+
+             StringBuilder result_string = new StringBuilder();
+            string header = "";
+            foreach (DataColumn column in results.Columns)
+            {
+                header += column.ColumnName + "\t";
+            }
+            result_string.AppendLine(header);
+            foreach (DataRow row in results.Rows)
+            {
+                result_string.AppendLine(String.Join("\t", row.ItemArray));
+            }
             return results;
         }
 
-        public static DataTable biological_replicate_intensities(IGoAnalysis analysis, IEnumerable<ExperimentalProteoform> proteoforms, List<InputFile> input_files, Dictionary<string, List<string>> conditionsBioReps, bool include_imputation)
+
+        public static DataTable topdown_results_dataframe()
+        {
+            DataTable results = new DataTable();
+            results.Columns.Add("PFR", typeof(string));
```

Comment:
Maybe "PFR Accession," instead? It's an accession, right?

Reply:
that's more clear!